iam_groups_authn/utils.py

```diff
@@ -33,3 +34,33 @@ async def run(*args, loop=None, executor=None, **kwargs):
         return await loop.run_in_executor(executor, pfunc)
 
     return run
+
+
+class DatabaseVersion(Enum):
+    """Enum class for database version."""
+
+    MYSQL_8_0 = "mysql"
+    POSTGRES_13 = "postgresql"
+    POSTGRES_12 = "postgresql"
+    POSTGRES_11 = "postgresql"
+    POSTGRES_10 = "postgresql"
+    POSTGRES_9_6 = "postgresql"
+
```

Comment:
These should probably have individual values corresponding to the value from the SQL Admin API. 

Then you can do something like:
```
db_version = DatabaseVersion("MYSQL_8_0")
```
 and get the proper enum. You can use a try/catch here to avoid needing to have a list of supported instances. 
 
 Finally, you can add a method like "is_database_type" or "is_mysql" and "is_postgres" to determine if the value starts with MYSQL or POSTGRES